### ISSUE TITLE
We need a prototype for displaying / moving things

### ISSUE THREAD

**Issue by Trevoke** [thoughtbot/write-yourself-a-roguelike#39]
We need a prototype of "there's a character on the screen and it moves". 

We may want to leverage the gosu library, though I know that would be a switch from what we currently have, so it may not be ideal:

- https://www.confreaks.tv/videos/madisonruby2013-rapid-game-prototyping-with-ruby 
- https://www.youtube.com/watch?v=jJhbpY70miE
- https://www.youtube.com/watch?v=L4RWsE8z-is

Whether we use gosu or stick to plain terminal, we need this. This means we need a volunteer. Who's going to put in the time for this?

**Comment by ngscheurich** [comment]
I will volunteer to get this underway. I vote for sticking with terminal + ncurses.

**Comment by tvanderpol** [comment]
I agree with keeping the dependencies very light - the whole point is to teach after all,  not to build an entire, compelling game.

**Comment by Trevoke** [comment]
Entirely reasonable :D

**Comment by halogenandtoast** [comment]
My original intent was just to use curses for this.

**Comment by Trevoke** [comment]
I think that's a great idea. Besides, knowing how to use curses leads to
being able to create/manage/test some terminal-based apps, so that has tons
of practical consequences :)
